m4_sessions/1808-inning/Perspective_to_succeed

```diff
@@ -1,19 +1,63 @@
 ## Group Members:
+john livingston
+spenser leighton
+arram mandel
 
 
-
-## Name of Session: 
+## Name of Session:
+disrupting your perspective: become an (secret) agent of disruption
 
 ## Session Description 
 
 (Why should students come to this session): Are you interested in biological enhancement? Drop in to learn about biohacking.
 
-## Session Outcomes 
+## Session Outcomes
+
+quote ghandi and yoda
+
+sell the benefit of disrupting your perspective, not pushing a certain perspective
+
+
 
 (What will students learn/leave with?): 
 
 ## Session Outline 
 
+
+
+count attendees as they walk into room, divide them into groups that way immediately!!!
+
+who do you like to watch lose?(10 min)
+
+introduce ourselves and something that makes us uncomfortable(3 min)
+
+have attendees talk about something that makes them uncomfortable in their groups. everyone will share.(10 min)
+
+show short clip of someone else talking about it(5 min)
+
+go into potential positives of disrupting perspective(10 min)
+
+-sparks innovation
+
+-causes growth(thor rag scene?)
+
+-Can contribute to empathy(or make you realize you don't understand)
+
+-Bring up confucian idea of no true self(https://www.theguardian.com/books/2016/apr/09/forget-mindfulness-stop-trying-to-find-yourself-start-faking-it-confucius)
+
+-Bring up agent of disruption article(https://medium.com/@johnsonwhitney/disrupt-yourself-5-ways-to-become-your-own-agent-of-disruption-77d7e2a9d16c)
+
+exercises for disrupting perspective(15 min)
```

Comment:
I really like this article!  And I think it would be SUPER helpful to have students spend 10 minutes discussing the article and then coming up with 3-5 ways they could start to do this at Turing. Ex: Sit in a different spot in class every day!  (Students get so uncomfortable doing this)